kubetest/extract.go

```diff
@@ -393,7 +393,8 @@ func (e extractStrategy) Extract(project, zone string) error {
 		}
 		return getKube(url, release)
 	case gcs:
-		return getKube(path.Dir(e.option), path.Base(e.option))
+		url := strings.Replace(path.Dir(e.option), "gs:/", "https://storage.googleapis.com/", 1)
```

Comment:
relying on `path.Dir` changing `gs://foo` to `gs:/foo` seems rather fragile (and also confusing - I didn't understand why this worked until I read the PR description).
maybe strip the leading `gs://`, call `path.Dir`, then prepend the https url?

Reply:
SGTM.

Reply:
Done.